### PR TITLE
Feature/setup travis ci

### PR DESCRIPTION
**Pull request by DrewBradley**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Create and setup `.travis.yml` file.
Configure `package.json` to include `scripts` found in `.yml` file.
Set up `cypess.json` to include `projectID`.

## *Instructions for reviewer:*
*Where should the reviewer start?*
Lines 22-28 of `package.json`
Lines 1-23 of `src/.travis.yml`
`cypress.json`
`home.spec.js`

*How should this be manually tested?*
NA
*Any background context you want to provide?*
To set up Travis with Cypress, a projectID and a key to allow Travis to record.
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

**Review by kevinhartmann23 — APPROVED** [review]
- [x] Changes generate no new warnings or errors
- [ ] Branch was tested locally without failures
- [ ] All tests are passing as expected

GREEN LIGHT 🪀

File: src/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    - ~/.cache
+  override:
+    - npm ci
+    - npm run cy:verify
+install:
+  - npm ci
+script:
+  - npm start &
+  - npm run start:ci &
+  - cypress run --record --key 86ce623b-9bbc-4993-8dca-f3535c528333
```
Comment (kristenmb) [inline src/.travis.yml:23]: I believe we will add a deploy key here as well which is where we will put vercel

**Review by kristenmb — APPROVED** [review]

File: src/.travis.yml (hunk 1 above)
Comment (DrewBradley) [reply src/.travis.yml:23]: Thanks fam!

**Review by richardltyler — APPROVED** [review]
yeet

**Pull request merged**